src/cascadia/TerminalSettingsEditor/Actions.xaml

```diff
@@ -0,0 +1,189 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<Page
+    x:Class="Microsoft.Terminal.Settings.Editor.Actions"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:SettingsModel="using:Microsoft.Terminal.Settings.Model"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="CommonResources.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+
+            <local:StringIsEmptyConverter x:Key="CommandKeyChordVisibilityConverter"/>
+
+            <!-- Template for actions. This is _heavily_ copied from the command
+            palette, with modifications:
+                * We don't need to use a HighlightedTextControl, because we're
+                  not filtering this list
+                * We don't need the chevron for nested commands
+                * We're not displaying the icon

```

Comment:
Wait... we should definitely handle these cases though. We're not showing the user what's in their settings.

At the very least, we should update the disclaimer to say that these are omitted right (regarding nested commands)?

Reply:
I'm gonna punt this to "have a real page for actions / commands". This is literally just the "do it quick and dirty" version.

I'll see if adding the icon is a pain though. That might be slick.

Reply:
Yeah, at least the icons would be nice to have haha